extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -120,31 +120,14 @@
 
     @ConfigGroup
     public static class Tls {
-        public enum Verification {
-            /**
-             * Certificates are validated and hostname verification is enabled. This is the default value.
-             */
-            REQUIRED,
-            /**
-             * All certificated are trusted and hostname verification is disabled.
-             */
-            NONE
-        }
 
         /**
-         * Certificate validation and hostname verification, which can be one of the following values from enum
-         * {@link Verification}. Default is required.
+         * Enable or disable certificate validation and hostname verification. Enable by default.
+         * 
+         * @deprecated use quarkus.tls.trust-all instead
```

Comment:
I'm not sure it should be deprecated. There has to be an option for each extension to choose to differ from the global value. The example I suggested during the forum discussion: Vault in internal, OIDC server is not.
So I'd rename `quarkus.oidc.tls.verification` to `quarkus.oidc.tls.trust-all` (initialized by default to `quarkus.tls.trust-all`) without the deprecated tag (and similarly for the other extensions). We can then update the migration guide. And docs to recommend `quarkus.tls.trust-all` but the users would still have an option to override it with `quarkus.oidc.tls.trust-all' for OIDC or `quarkus.vault.tls.trust-all' for Vault, etc...